WalletWasabi/Helpers/RuntimeParams.cs

```diff
@@ -82,7 +82,8 @@ public static async Task LoadAsync()
 			}
 
 			string jsonString = await File.ReadAllTextAsync(FilePath, Encoding.UTF8).ConfigureAwait(false);
-			InternalInstance = JsonConvert.DeserializeObject<RuntimeParams>(jsonString);
+			InternalInstance = JsonConvert.DeserializeObject<RuntimeParams>(jsonString)
+							?? throw new InvalidOperationException($"Couldn't deserialize {nameof(InternalInstance)} from {FilePath}.");
```

Comment:
IMO we can improve the exc message.
```suggestion
							?? throw new InvalidOperationException($"Couldn't deserialize {typeof(InternalInstance)} from {FilePath}.");
```
Yours:

`Couldn't deserialize InternalInstance from "/../../..".`

Suggestion:
`Couldn't deserialize RuntimeParams from "/../../..".`